index.html

```diff
@@ -5,19 +5,24 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Personal Portfolio</title>
+    <link rel="stylesheet" href="css/index.css">
 </head>
 <body>
     <h1>Haregebezu Amare</h1>
-    <section>
-        <h2>About</h2>
+    <section class="about">
+        <h2>About</h2> 
             <p>I have immersed myself in the world of software development for over a year. With an insatiable curiosity and passion
             for technology, I have delved into various programming languages, frameworks, and tools, building a solid foundation of
             technical knowledge. From mastering web development with HTML, CSS, and JavaScript, I have developed a versatile
             skill set. I am committed to lifelong learning, staying up to date with industry trends through workshops and online
             courses. Each day, embrace the challenges and rewards of being a software developer, poised to make a significant impact
             in the world of technology. </p>
-        <img src="nice wprk place.jpg" alt = "working place" style="max-width: 800px">  
-        <h2>Experience</h2>
+    </section>
+    <section class="image">
+        <img src="nice wprk place.jpg" alt = "working place" style="width: 800px;" > 
+    </section>
+    <section class="exprience">
+        <h2 >Experience</h2>
```

Comment:
```suggestion
        <h2>Experience</h2>
```
Consistent space, just to keep it tidy.